service/history/historyEngine.go

```diff
@@ -380,7 +380,7 @@ func (e *historyEngineImpl) StartWorkflowExecution(startRequest *h.StartWorkflow
 	}
 
 	if err == nil {
-		e.timerProcessor.NotifyNewTimers(timerTasks)
+		e.timerProcessor.NotifyNewTimers(e.shard.GetService().GetClusterMetadata().GetCurrentClusterName(), timerTasks)
```

Comment:
Can we just store the current cluster name on the historyEngine instead of getting each time from Cluster Metadata.

Reply:
sure